FWCore/Framework/interface/EventSetupProvider.h

```diff
@@ -92,7 +93,8 @@ namespace edm {
           EventSetupProvider& precedingESProvider,
           std::set<ParameterSetIDHolder>& sharingCheckDone,
           std::map<EventSetupRecordKey, std::vector<ComponentDescription const*>>& referencedESProducers,
-          EventSetupsController& esController);
+          EventSetupsController& esController,
+          ModuleTypeResolverBase const* resolver);
```

Comment:
I'd prefer having this moved to the first argument as I normally place mutable arguments at the end of function call argument lists.

Reply:
Changed the order. Also in another function where it seemed to make sense.